core/modules/widgets/checkbox.js

```diff
@@ -3,129 +3,134 @@ title: $:/core/modules/widgets/checkbox.js
 type: application/javascript
 module-type: widget
 
-Checkbox widget
+Checkbox widget to toggle a tag, or set/reset the value of a field or index
 
 \*/
-(function(){
+(function() {
 
 /*jslint node: true, browser: true */
 /*global $tw: false */
 "use strict";
-
 var Widget = require("$:/core/modules/widgets/widget.js").widget;
 
-var CheckboxWidget = function(parseTreeNode,options) {
-	this.initialise(parseTreeNode,options);
-};
-
 /*
 Inherit from the base widget class
 */
+var CheckboxWidget = function(parseTreeNode, options) {
+	this.initialise(parseTreeNode, options);
+};
+
 CheckboxWidget.prototype = new Widget();
 
 /*
 Render this widget into the DOM
 */
-CheckboxWidget.prototype.render = function(parent,nextSibling) {
-	// Save the parent dom node
+CheckboxWidget.prototype.render = function(parent, nextSibling) {
 	this.parentDomNode = parent;
-	// Compute our attributes
 	this.computeAttributes();
-	// Execute our logic
 	this.execute();
-	// Create our elements
 	this.labelDomNode = this.document.createElement("label");
-	this.labelDomNode.setAttribute("class",this.checkboxClass);
+	this.labelDomNode.setAttribute("class", this.checkboxClass);
```

Comment:
Mhhh, when I edit the core with [brackets](http://brackets.io/) or [sublime text](http://www.sublimetext.com/), it appears that my formatting turns out to conform to convention.

I never do any automagic beautification because that never appears to work quite right.

> all of the formatting libraries leave a space after the comma between parameters (looks like this is standard.)

Interesting, it's what I did at the beginning as well.

> Manual editing isn't a good technique when trying standardise on code formatting -- so, I don't go there.

Not sure, it's how you learn it. If there was some tw-core-dev-format that can be defined and then have some beautification plugin apply that, that would be best. Looks like it's not there... and also not quite formalized either, I presume.